### DISCUSSION TITLE
Offical Extension Store

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#70] [Feedback]
I think a official extension store for Editor Mode Extensions would be a great idea. I think a similar implementation to blockbench's Plugins would be best. There can be the official extensions by Mojang themselves and maybe even community ones that have been approved. I don't think these extensions should cost any money.

![Blockbench 3_10_2023 11_04_10 PM](https://user-images.githubusercontent.com/71415477/224463942-df210ba6-9311-40ff-835a-622e98ee028a.png)
![Blockbench 3_10_2023 11_07_52 PM](https://user-images.githubusercontent.com/71415477/224464068-5995233a-7c9c-44e5-a66e-69c39e875a6b.png)

**Comment by CaptainCooky** [comment]
Great Idea, having one place to get/install extensions would easily the best way to ensure good workflow and quality. 

VSCode also has an extension store similar to that of Blockbench.

**Comment by minecraftchloe** [comment]
+1 this is an incredible idea ;)
